<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCalibrationAgr1</class>
 <widget class="QDialog" name="SelectCalibrationAgr1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выбор калибровки</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="Time">
      <attribute name="title">
       <string>Time</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_84">
           <property name="text">
            <string>Реагенты, серия №</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_17"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_85">
           <property name="text">
            <string>Годен до:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_16"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Время инкубации</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ACHTV">
      <attribute name="title">
       <string>ACHTV</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Последняя калибровка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_calibrationData_3">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Реагенты, серия №</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_18</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_18"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Годен до</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_19</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_19"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>К/плазма, серия №</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_20</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_20"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>К/плазма до</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_21</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_21"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>АЧТВ К/плазмы</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_22</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_22"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Время инкубации</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Trombin">
      <attribute name="title">
       <string>Trombin</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_50">
           <property name="text">
            <string>Последняя калибровка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_calibrationData_6">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_48">
           <property name="text">
            <string>Реагенты, серия №</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_23"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_52">
           <property name="text">
            <string>Годен до</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_24"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_49">
           <property name="text">
            <string>К/плазма, серия №</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_25"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_53">
           <property name="text">
            <string>К/плазма до</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_26"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_51">
           <property name="text">
            <string>Тромбин К/плазмы</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_27"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_47">
           <property name="text">
            <string>Время инкубации</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Fibrinogen">
      <attribute name="title">
       <string>Fibrinogen</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_54">
           <property name="text">
            <string>Последняя калибровка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_calibrationData_7">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_56">
           <property name="text">
            <string>Реагенты, серия №</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_28"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Годен до</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_29"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_57">
           <property name="text">
            <string>Фибрмноген к/плазмы</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_30"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_58">
           <property name="text">
            <string>Время свертывания при</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_31"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_59">
           <property name="text">
            <string>Разведение 200%:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_32"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_60">
           <property name="text">
            <string>Разведение 100%:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_33"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_55">
           <property name="text">
            <string>Разведение 50%:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_34"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_63">
           <property name="text">
            <string>Разведение 25%:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit_35"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_61">
           <property name="text">
            <string>Время инкубация:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Protrombin">
      <attribute name="title">
       <string>Protrombin</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_94">
           <property name="text">
            <string>Последняя калибровка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_42"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_96">
           <property name="text">
            <string>Тромбопластин, серия №</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_36"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_93">
           <property name="text">
            <string>Годен до</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_37"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_97">
           <property name="text">
            <string>К/плазма, серия №</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_38"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_98">
           <property name="text">
            <string>К/плазма</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_43"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_101">
           <property name="text">
            <string>Разведение 100%:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_44"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_100">
           <property name="text">
            <string>Разведение 50%:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_45"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_95">
           <property name="text">
            <string>Разведение 25%:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_46"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_102">
           <property name="text">
            <string>Разведение 12,5%:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit_47"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_99">
           <property name="text">
            <string>Время инкубация:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ParamAGR">
      <attribute name="title">
       <string>ParamAGR</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_88">
           <property name="text">
            <string>Реагенты, серия №</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_39"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_89">
           <property name="text">
            <string>Годен до:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_40"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_91">
           <property name="text">
            <string>Концентрация тромбинов</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_41"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_90">
           <property name="text">
            <string>Время инкубации</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Villebrand">
      <attribute name="title">
       <string>Villebrand</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>Последняя калибровка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_calibrationData">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Реагенты, серия №</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Годен до:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Разведение 15%:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_15"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Разведение 30%:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Разведение 60%:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_11"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Разведение 75%:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_12"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Разведение 100%:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_13"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Количество точек калибровки:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Время инкубация, сек:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_apply">
        <property name="text">
         <string>Применить калибровку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_noCalibration">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Без калибровки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_newCalibration">
        <property name="text">
         <string>Новая калибровка</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>pushButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>radioButton_newCalibration</tabstop>
  <tabstop>radioButton_noCalibration</tabstop>
  <tabstop>radioButton_apply</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectCalibrationAgr1</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
